### ISSUE TITLE
news - date drop down - javascript jump menu?

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#103]
WAVE is saying there may be a javascript jump menu present on the drop down at the far right of the page. This is what it's saying about it:

![Screen Shot 2020-10-06 at 4 18 17 PM](https://user-images.githubusercontent.com/618307/95256043-45d40380-07f0-11eb-941a-b0d0c911dfc5.png)

**Comment by kimisgold** [comment]
Just added a submit button to the select navigation form that's ready to test.

**Comment by IdaAkiwumi** [comment]
I'm unable to get to the GO button after selecting the month on the main news page: https://rrchnm.org/news/. When I select the month in the dropdown, it goes to the actual url of that month before I get to see and use the button. On the actual individual month pages, it behaves as expected. 

It still shows a JavaScript jump menu issue; however, the few individual month pages I tested don't flag that issue.

**Comment by kimisgold** [comment]
Ah okay, this had to be applied to both the archives view and the index. It should be consistent now.
